src/System.Collections/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -249,7 +249,7 @@ private void CopyTo(KeyValuePair<TKey, TValue>[] array, int index)
 
             if (index < 0 || index > array.Length)
             {
-                throw new ArgumentOutOfRangeException("index", SR.ArgumentOutOfRange_NeedNonNegNum);
+                throw new ArgumentOutOfRangeException("index", index, SR.ArgumentOutOfRange_NeedNonNegNum);
```

Comment:
Whoops - I found and changed several exception messages like this but this one slipped past my radar. I'll change it soon